src/Adapters/EventSource.Net46/EventSourceListeningRequest.cs

```diff
@@ -0,0 +1,72 @@
+﻿// -----------------------------------------------------------------------
```

Comment:
OK, I will add them back to the files in this commit, but for other files in the repo, that should be taken care of as a separate work item